mappings/net/minecraft/state/State.mapping

```diff
@@ -26,7 +26,7 @@ CLASS net/minecraft/class_2688 net/minecraft/state/State
 	METHOD method_28499 toMapWith (Lnet/minecraft/class_2769;Ljava/lang/Comparable;)Ljava/util/Map;
 		ARG 1 property
 		ARG 2 value
-	METHOD method_28500 (Lnet/minecraft/class_2769;)Ljava/util/Optional;
+	METHOD method_28500 getOrNone (Lnet/minecraft/class_2769;)Ljava/util/Optional;
```

Comment:
```suggestion
	METHOD method_28500 getOrEmpty (Lnet/minecraft/class_2769;)Ljava/util/Optional;
```
Since the absent `Optional` is called `Optional.empty`